### PR TITLE
Ft user

### PR DESCRIPTION
**Pull request by ernesthenry**

### CODE REVIEW

**Comment by llwasampijja** [comment]
@ernesthenry The best way to deal solve a merge conflict problem is to avoid it in the first place and that is by following a proper git work flow. When merging, always make sure that the branch with the new commits has all the commits in the other branch!
That said, you can use vs code to view the merge conflicts and decide which code changes you want to keep or not!

File: api/routes.py [hunk 1]
```diff
@@ -18,8 +22,9 @@ def create_redflag():
             "status": 400
             }), 400
     data = request.get_json()
-    if 'createdBy' not in data or 'comment' not in data or 'type' not in data or 'location' not in data or 'status' not in data:
```
Comment (BrunoNicholas) [inline api/routes.py]: this is the update I think so you can look through the merge conflicts here in this text editor and remove the old  conflicting lines of code directly and then merge after

**Comment by ernesthenry** [comment]
okay. let me do it

File: api/routes.py (hunk 1 above)
Comment (ernesthenry) [reply api/routes.py]: Sorry for responding a bit late. But i have never resolved conflicts onlne

**Pull request merged**